### ISSUE TITLE
Big one - optional - Manage lists on WordPress back-end?

### ISSUE THREAD

**Issue by thisisjoewells** [TheDetroitZoo/pr-mailchimp-integration#25]
Last one from the meeting with Patricia, Jennie, and CJ...

They liked everything they saw.  The only thing they were confused and -I suppose- a bit disappointed about was the managing of the Media list.

I told them they could easily edit the list easily at MailChimp.

They said they actually edit the Media list every week or two.  They said "we thought we were going to be able to do _everything_ on WordPress, isn't that the point?".

So.  Let's discuss.  Note that I did not mile-stone this as "PR/Comm Prototype Revisions".  This is the one I mentioned yesterday as kinda bit and possibly a future iteration or improvement if it's a complex build.

Could there be, should there be a way to manage the Media list somehow on the back-end of WP?  Is this crazy?  Could we just have a link on the back-end that is tokenized kinda like user-access-keys for Polk/Parking Alerts that takes them to a MailChimp list management page?  Who knows....

**Comment by d4mation** [comment]
It would be possible, but I honestly think editing it from MailChimp itself would be better. 

It would be a neat feature to manage the List from WordPress, but we'd probably want to dedicate an entire page to it (So as not to add more clutter to the Campaign Sending form) and at that point you may as well visit the MailChimp site.

As far as implementation goes, I can see something along the lines of:

- Multi-select field for deleting Members of a List. Choose one or more and then hit a delete button.
- Simple AJAX Form for adding to a List. 

There are other things that could be done for a List, but I figure those are the main actions that would be needed, right?

**Comment by thisisjoewells** [comment]
I agree that one may as well go to MailChimp, once we build an interface on it's own WP back-end page, it will likely be inferior and not actually save time.

Yes, I think if we did have some sort of interface just for deleting list members, that may take care of it.  For now, let me see if we can launch it, let them use it, and see what need arises.

Quick Fix/Improvement Idea:
Perhaps we just make a prominent link on both the current settings page and at the bottom of the 4-column interface that at least goes to the MailChimp list management page.  I just tried this.  I was not logged in, went to this link, MC made me log in then sent me to the list page.  This would make things easier for them, everything KINDA happens in WordPress.

Link - - https://us1.admin.mailchimp.com/lists/

This would work as long as our serer (us1) doesn't change.  :)

Another thought, but we need not move on this at all now...

Let's say they need to occasionally tweak the list a bit, add a person, remove 3 or 4, just for one Press Release.  And I'm not saying they need to.  I think their concern is just easily removing people.  But we could use List Segments, right?  So perhaps a future-add-on --but not now, until they use the system and decide on how it will be used-- would be to have a segment drop-down that appears below the list if they select a list that has segments.

Thoughts?

**Comment by d4mation** [comment]
From what I understand about Segments, they only consist of Users within a List that match certain criteria. Generally grouped by shared meta data. However, they can be created as a Static Segment to hold whatever individual e-mails that we want.

We could potentially add/remove based on Segments if those 3 or 4 Users are always the same.

**Comment by brashrebel** [comment]
The segment or group selection sounds like a very logical future enhancement to me.

**Comment by thisisjoewells** [comment]
Ok.  Here is the verdict for now.  We wait on possibly adding Segments and/or another way to manage lists from WordPress for some future enhancement.

But as a Stop-Gap, to make it feel to end-user that "everything" happens from "WordPress" can we:

1 - make a link that says 'Edit/Manage Lists' going to (_blank link) - - https://us1.admin.mailchimp.com/lists/ - - at the top of the 4-column interface and 
2 - right after the Boilerplate on the main settings page (also _blank link)

A bold text link should suffice.  I'll make a separate issue for this and leave this issue pending-DZS.
